src/main/java/org/generationcp/middleware/domain/dms/PhenotypicType.java

```diff
@@ -28,14 +28,16 @@ public enum PhenotypicType {
 			"SITENO"), "TRIAL ENVIRONMENT"), //
 
 	GERMPLASM(Arrays.asList(TermId.GERMPLASM_ENTRY_STORAGE.getId(), TermId.ENTRY_NUMBER_STORAGE.getId(), TermId.ENTRY_GID_STORAGE.getId(),
-			TermId.ENTRY_DESIGNATION_STORAGE.getId(), TermId.ENTRY_CODE_STORAGE.getId()), Arrays.asList("ENTRY", "ENTRY_NO", "ENTRYNO"),
-			"GERMPLASM ENTRY"), //
+			TermId.ENTRY_DESIGNATION_STORAGE.getId(), TermId.ENTRY_CODE_STORAGE.getId()), Arrays.asList(""), "GERMPLASM ENTRY"), //
+
+	ENTRY_DETAIL(new ArrayList<>(), Arrays.asList("ENTRY", "ENTRY_NO", "ENTRYNO"), "ENTRY_DETAIL"),
 
 	TRIAL_DESIGN(Arrays.asList(TermId.TRIAL_DESIGN_INFO_STORAGE.getId()), Arrays.asList("PLOT", "PLOT_NO", "PLOTNO"), "TRIAL DESIGN"), //
 
 	VARIATE(Arrays.asList(TermId.OBSERVATION_VARIATE.getId(), TermId.CATEGORICAL_VARIATE.getId()), Arrays.asList("STUDY", "PLOT", "TRIAL"),
 			"VARIATE"),
 
+	// TODO: should be labelList a list with an empty string just like DATASET?
```

Comment:
@lik-83 @cuenyad please take a look at this TODO and let me know if there is anything we would like to do in this branch

Reply:
There is anything to do with this TODO. So, I removed the comment.